pyccel/ast/mathext.py

```diff
@@ -106,91 +106,271 @@ class MathFunctionBool(MathFunctionBase):
 
 # Floating-point result
 class MathAcos    (MathFunctionFloat):
-    """Represent a call to the acos function in the Math library"""
+    """
+    Class representing a call to the `math.acos` function.
+
+    A class which represents a call to the `acos` function from the `math` library.
+
+    Parameters
+    ----------
+    x : PyccelAstNode
+        The expression passed as argument to the function.
+    """
     __slots__ = ()
     name = 'acos'
 class MathAcosh   (MathFunctionFloat):
-    """Represent a call to the acosh function in the Math library"""
+    """
+    Class representing a call to the `math.acosh` function.
+
+    A class which represents a call to the `acosh` function from the `math` library.
+
+    Parameters
+    ----------
+    x : PyccelAstNode
+        The expression passed as argument to the function.
+    """
     __slots__ = ()
     name = 'acosh'
 class MathAsin    (MathFunctionFloat):
```

Comment:
Thanks for adding the docstrings! Given that the code is not as compact as before, it is worth formatting it properly. Here it should be sufficient to add two empty lines between the classes.

Reply:
Fixed f025067b